Sampled/UI/VIews/LibraryView.swift

```diff
@@ -759,25 +798,55 @@ struct LibraryView: View {
         }
       }
     }
-    .toolbar {
-      ToolbarItem(id: "\(Bundle.appID).library-up-next") {
-        // TODO: Localize.
-        Button("Inspect", systemImage: "list.number") {
-          isInspectorPresented.toggle()
+      .toolbar {
+        ToolbarItem(id: "\(Bundle.appID).library-up-next") {
+          // TODO: Localize.
+          Button("Inspect", systemImage: "list.number") {
+            isInspectorPresented.toggle()
+          }
+        }
+        
+        ToolbarItem(id: "\(Bundle.appID).history") {
+          Button {
+            NSWorkspace.shared.open(URL(string: "sampled://history")!)
```

Comment:
We don't need our own protocol for this. You should be able to use [openWindow](https://developer.apple.com/documentation/swiftui/environmentvalues/openwindow) for this.